### DISCUSSION TITLE
Enable searching pull requests by resolved/unresolved comments

### DISCUSSION THREAD

**Discussion by weitzhandler** [community/community#64078] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I'm working on a repo and periodically I would like to take care of outstanding PRs, this can be either:

- Searching which PRs have unresolved comments that I need to work on.
- Search which PRs don't have any unresolved comments that are ready to be merged.

Please add a search operator something like `review:has_comments` / `review:no_comments` or similar.

**Comment by andreahlert** [comment]
Hi 🙂
This would be a big productivity win. On active repositories it’s hard to know which PRs are truly “clean” versus those with lingering discussions. Being able to filter by unresolved comments would help reviewers and maintainers prioritize faster and reduce review fatigue.

Related feature that partially helps today:
https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests
